azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/entity/AppServicePlanEntity.java

```diff
@@ -7,17 +7,25 @@
 import com.microsoft.azure.toolkit.lib.appservice.model.OperatingSystem;
 import com.microsoft.azure.toolkit.lib.appservice.model.PricingTier;
 import com.microsoft.azure.toolkit.lib.common.entity.IAzureResourceEntity;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
 @Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
 @SuperBuilder(toBuilder = true)
 public class AppServicePlanEntity implements IAzureResourceEntity {
-    private final String subscriptionId;
-    private final String id;
-    private final String name;
-    private final String region;
-    private final String resourceGroup;
-    private final PricingTier pricingTier;
-    private final OperatingSystem operatingSystem;
+    private String subscriptionId;
+    private String id;
+    private String name;
+    private String region;
+    private String resourceGroup;
+    private PricingTier pricingTier;
+    private OperatingSystem operatingSystem;
```

Comment:
is there somewhere use setXXX ？It seems "No" in this PR.

Reply:
Not here, but xml serialization/deserialization in IntelliJ run configuration